### PR TITLE
fix docker tls

### PR DESCRIPTION
**Pull request by vtolstov**
if user provide tls certs we need use it in all cases

Signed-off-by: Vasiliy Tolstov <v.tolstov@selfip.ru>

### CODE REVIEW

File: pkg/runner/builtin/runner.go [hunk 1]
```diff
@@ -240,7 +240,27 @@ func (r *Runner) Run(w *queue.Work) error {
 }
 
 func (r *Runner) Cancel(job *types.Job) error {
-	client, err := dockerclient.NewDockerClient(DockerHost, nil)
+	var tlc *tls.Config
```
Comment (Bugagazavr) [inline pkg/runner/builtin/runner.go]: this code repeated twice, maybe you can move that to one function?

Comment (vtolstov) [inline pkg/runner/builtin/runner.go]: I think about this (3 times repeated =)), may be we need something like newDockerClient() function...

**Comment by vtolstov** [comment]
@Bugagazavr i'm fix pr.

**Comment by tboerger** [comment]
:+1:

**Comment by bradrydzewski** [comment]
thanks!

**Pull request merged**